js/interface.js

```diff
@@ -0,0 +1,64 @@
+var appDataSources = [];
+Fliplet.DataSources.get({ appId: Fliplet.Env.get('masterAppId'), attributes: ['id', 'name'] }).then(function (DSs) {
```

Comment:
You should set up https://github.com/Fliplet/eslint-config-client so your code is automatically linted to match our coding standards.

```suggestion

Fliplet.DataSources.get({ appId: Fliplet.Env.get('masterAppId'), attributes: ['id', 'name'] }).then(function (DSs) {
```

Reply:
I would need some help setting this up. Thanks